client/containers/DashboardSubmissionWorkflow/SubmissionWorkflowStepPicker.tsx

```diff
@@ -51,7 +51,6 @@ const SubmissionWorkflowStepPicker = (props: Props) => {
 							reached && 'reached',
 							incomplete && 'incomplete',
 						)}
-						stopId={step}
```

Comment:
This prop no longer exist on the reakit `Tab` component. I have tested the SW stepper and it seems non breaking